src/tables.css

```diff
@@ -31,7 +31,7 @@
  *
  */
 .table,
-.prose table {
+.prose table:not(.unprose) {
```

Comment:
Whoa the :not select doesn't work the way I thought it worked, but that makes sense. I prefer the idea of unprose containers, because it mirrors prose containers. Would you be opposed to using only `prose table:not(.unprose table)`?